app/assets/stylesheets/media.scss

```diff
@@ -426,17 +426,29 @@
       display: flex;
       flex-direction: column;
       overflow: clip auto;
+
       section {
         // This avoids wrapping when animating.
         width: calc(var(--drawer-width) - var(--tab-width));
       }
     }
 
-    .metadataList, .rightsList, .contentsList, .transcript, .autoscroll {
+    .metadataList, .rightsList, .contentsList, .transcript, .transcript-controls {
       color: var(--text-color);
       padding: var(--drawer-content-padding);
     }
 
+    .transcript-controls {
+      font-size: 0.8rem;
+
+      input, select {
+        margin-left: 0.6rem;
+        margin-right: 0.6rem;
+        vertical-align: middle;
+      }
+    }
```

Comment:
Per above, make header text larger than content text.

Also, try to get the spacing between the auto scroll label, checkbox, and language select dropdown closer to the design.